fizzbuuzz.js

```diff
@@ -0,0 +1,27 @@
+
+
+function fizzbuzz(Fizz,Buzz){
```

Comment:
Good work on this but I don't think that this function does what it is supposed to do.
Here is what I think is wrong
1. The function should just return 'Fizz', 'Buzz', or 'FizzBuzz'  but you are just logging them out. You can learn about returning a value from a function at [w3schools](https://www.w3schools.com/js/js_functions.asp) or [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions)
Below is a simple than demonstrates returning a value.
```js
// is function takes a number and return 'ODD' if number is odd and 'EVEN' otherwise
function isOddOrEven(number){
 // id number id odd
 if(number%2 === 1){
  // When a function encounters a return  statement, It stops executing 
  // so if we return 'ODD' code below the `return 'ODD';` line won't execute
   return 'ODD';
 }
 // if the number is not odd we return 'EVEN'
 return 'EVEN'
}

// using our function 
console.log(isOddOrEven(2)) // logs 'EVEN' to the screen
console.log(isOddOrEven(3)) // logs 'ODD' to the screen
```
2. The condition that checks for a number being divisible by 3 and 5 should come first and conditions below it shouldn't run if its true but here it is coming last. This means conditions before it will be true and code in them will be executed which we don't want

Reply:
@dennisja resolved on second commit